docs/backend/schema.ddl

```diff
@@ -0,0 +1,115 @@
+///////////////////////////////////////////////////
+// DPHs and users
+
+Table Dph {
+  id int [pk, increment]
+  name varchar [not null, unique]
+  gisaid_division_name varchar [unique]
+}
+
+Table User {
+  id int [pk, increment]
+  name varchar [not null]
+  auth0_user_id varchar [not null]
+  dph int [not null, ref: > Dph.id]
+  dph_admin bool [not null]
+}
+
+// members of dph can see data uploaded by owner_dph
+Table DphCanSee {
+  dph int [not null, unique, ref: > Dph.id]
+  owner_dph int [not null, unique, ref: > Dph.id]
+}
```

Comment:
@sidneymbell  could you comment on this?

Reply:
what is the difference between owner_dph and dph_admin?

Reply:
I should rename that as:

viewing_dph
owner_dph

Meaning viewing_dph can see the data for owner_dph.

Reply:
I'm also not sure if we would need the `DphCanSee` table since we could determine whether a user was in a dph or an admin by looking at the `User` table directly.

Reply:
`DphCanSee` is for people to cross that boundary.

Reply:
sorry i'm not really sure what you mean by that. Why not just query the User and see what DPH they belong to (and if they are an admin) directly in the User table? If they are a part of the DPH then they can see the data?

Reply:
So for instance, we might have an entry for people in CADPH to view SCCDPH's data.

Reply:
ok got it thanks

Reply:
I'm not fully sold this is the right model, because it doesn't fully capture the nuances.  For instance, SCCDPH might want to let CADPH view its data but not its metadata.  But making it a full cross product might be a bit too heavyweight.